templates/project/README.md.twig

```diff
@@ -21,7 +21,9 @@ It's auto-generated by sonata-project/dev-kit package.
 
 Branch | Github Actions | Code Coverage |
 ------ | -------------- | ------------- |
+{% if project.stableBranch is not null %}
 {{ project.stableBranch.name }}    | [![Test][test_stable_badge]][test_stable_link]     | [![Coverage Status][coverage_stable_badge]][coverage_stable_link]     |
```

Comment:
You can remove the links at the bottom of the README too

Reply:
Done, and I found some occurence of `project.stableBranch` so I added some checks.